src/zope/file/tests/test_doctests.py

```diff
@@ -13,25 +13,37 @@
 """Unit tests for zope.file.
 
 """
-from __future__ import print_function, absolute_import, division
+
 
 __docformat__ = "reStructuredText"
 import doctest
 import unittest
 
 from zope.file import testing
 
+from zope.testing import renormalizing
+import re
+
+checker = renormalizing.RENormalizing([
+    # Python 3 unicode removed the "u".
+    (re.compile("u('.*?')"), r"\1"),
+    (re.compile('u(".*?")'), r"\1"),
+    # Python 3 bytes removed the "b".
```

Comment:
I think you mean either `Python 2 bytes doesn't have the "b"` or `Python 3 bytes added the "b"`.